mathics/core/parser/operators.py

```diff
@@ -17,6 +17,8 @@
     'Exists': 240,
     'NotExists': 240,
     'Not': 230,
+    'Information': 5001,
+    'Definition' : 5000,
```

Comment:
What's the correct precedence of these operators?

Reply:
Seems to work as it is... we can try to change it to behave more like the weird way Wolfram's does it in weird cases, but at a practical level, it seems that does not matter.

Reply:
It works as it is